examples/mod-source-record-manager/profile.sample

```diff
@@ -0,0 +1,5 @@
+{
+  "id": "1kl9m60r-fvb7-45n0-23c0-13n8gbkl7091",
+  "name": "Profile for bib files",
+  "state": "COMMITTED"
```

Comment:
we need state?

Reply:
yes, we need state for the Profile but not for JobExecutionProfile.